packages/dds/tree/src/shared-tree/sharedTree.ts

```diff
@@ -811,9 +813,9 @@ function exportSimpleNodeSchemaStored(schema: TreeNodeStoredSchema): SimpleNodeS
 		return { kind: NodeKind.Array, allowedTypes: arrayTypes };
 	}
 	if (schema instanceof ObjectNodeStoredSchema) {
-		const fields: Record<string, SimpleFieldSchema> = {};
-		for (const [key, field] of schema.objectNodeFields) {
-			fields[key] = exportSimpleFieldSchemaStored(field);
+		const fields: Map<FieldKey, SimpleObjectFieldSchema> = new Map();
```

Comment:
Nit: should we prefer the following (a bit more concise)?
```suggestion
		const fields = new Map<FieldKey, SimpleObjectFieldSchema>();
```